lib/sprockets/sassc_importer.rb

```diff
@@ -0,0 +1,186 @@
+require 'sprockets/autoload'
+
+module Sprockets
+  class SasscImporter < Autoload::SassC::Importer
+    class Extension
+      attr_reader :postfix
+
+      def initialize(postfix=nil)
+        @postfix = postfix
+      end
+
+      def import_for(full_path, parent_dir, options)
+        SassC::Importer::Import.new(full_path)
+      end
+    end
+
+    class CSSExtension
+      def postfix
+        ".css"
+      end
+
+      def import_for(full_path, parent_dir, options)
+        import_path = full_path.gsub(/\.css$/,"")
+        SassC::Importer::Import.new(import_path)
+      end
+    end
+
+    class CssScssExtension < Extension
+      def postfix
+        ".css.scss"
+      end
+
+      def import_for(full_path, parent_dir, options)
+        source = File.open(full_path, 'rb') { |f| f.read }
+        SassC::Importer::Import.new(full_path, source: source)
+      end
+    end
+
+    class CssSassExtension < Extension
+      def postfix
+        ".css.sass"
+      end
+
+      def import_for(full_path, parent_dir, options)
+        sass = File.open(full_path, 'rb') { |f| f.read }
+        parsed_scss = SassC::Sass2Scss.convert(sass)
+        SassC::Importer::Import.new(full_path, source: parsed_scss)
+      end
+    end
+
+    class SassERBExtension < Extension
+      def postfix
+        ".sass.erb"
```

Comment:
@rafaelfranca Based on http://guides.rubyonrails.org/asset_pipeline.html#preprocessing doesn’t Sprockets do this kind of thing by default?

(Full disclaimer: I don’t know Sprockets super well – I’ve mostly been working on the Sass side of things.)